src/components/UserHeaderCtrl/UserHeaderCtrl.tsx

```diff
@@ -58,7 +58,7 @@ class UserHeaderCtrl extends Component<Props, State> {
         userSetInfo({ subscribedUserIds: response.data })
       }
     } catch (error) {
-      if (error && error.response && error.response.data && error.response.data.message === 'Premium Membership Required') {
+      if (error && error.response && error.response.data && error.response.data.message === PV.errors.premiumRequired) {
```

Comment:
I'm going to move this error message into a separate constants file PV.errorResponseMessages.

`error.response.data.message` represents the error message sent back in an API response, and there's no guarantee that the error messages sent by the API will match text shown elsewhere in the web app.

Using `error.response.data.message` in an `if` is actually a bad practice. I should be checking for `error.response.statusCode` (ex. 404) and using that to decide how to handle the error sent back from the API. I'm not going to change it now though because this is working and I don't want to disturb it...